src/Symfony/Component/Messenger/Worker.php

```diff
@@ -92,6 +97,8 @@ public function run(array $options = []): void
         while (!$this->shouldStop) {
             $envelopeHandled = false;
             $envelopeHandledStart = microtime(true);
+            $receivers = $this->receivers;
+            $receivers = self::STRATEGY_RANDOM === $options['strategy'] ? shuffle($receivers) : $receivers;
```

Comment:
It looks like the variable `$receivers` is not used in the following code.

Reply:
Forgot to change the line below indeed, fixed it